doc/pairs.md

```diff
@@ -0,0 +1,205 @@
+# Pairs examples
+
+With ``crud.pairs``, you can iterate across a distributed space. The arguments are the same as [``crud.select``](https://github.com/tarantool/crud/docs/select.md), except or the ``use_tomap`` parameter. Below are examples that may help you.
+
+## Getting space
+
+Let's check ``developers`` space contents to make other examples more clear. Just select first 4 values without conditions.
+
+**Example:**
+
+```lua
+tuples = {}
+for _, tuple in crud.pairs('developers', nil, { first = 4 }) do
+  table.insert(tuples, tuple)
+end
+
+tuples
+---
+- - - 1
+    - 7331
+    - Alexey
+    - Adams
+    - 20
```

Comment:
It would be great to describe fields and schema of space briefly. I see no information above.